docs/README.md

```diff
@@ -59,6 +58,16 @@ The credentials can be supplied by providing the `modules.core.aws` setting in t
 }
 ```
 
+AWS SDK can be also configured in code by using `altis.aws_sdk.params` filter which is called right before the global instance of the AWS SDK is created.
+
+```
```

Comment:
We can use the GitHub style markdown to indicate the language being used in code snippets to use appropriate highlighting:
```suggestion
```php
```